server/sonar-server/src/test/java/org/sonar/server/db/DatabaseCheckerTest.java

```diff
@@ -70,14 +100,16 @@ public void log_warning_if_h2() throws Exception {
 
   @Test
   public void do_not_fail_if_mysql() throws Exception {
-    Database db = mockDb(new MySql(), "5.7");
+    Database db = mockDb(new MySql(), "5.7", "5.7");
     new DatabaseChecker(db).start();
     // no error
   }
 
-  private Database mockDb(Dialect dialect, String driverVersion) throws SQLException {
+  private Database mockDb(Dialect dialect, String dbVersion, String driverVersion) throws SQLException {
```

Comment:
Why not directly use an int for dbVersion? This would simplify mock initialization.